server/arweave/index.ts

```diff
@@ -150,15 +150,17 @@ export async function estimateARPrices(files: ArweaveFile[]): Promise<ArweavePri
   }
 }
 
-export function convertARPriceToLIKE(ar: ArweavePrice, {
-  priceRatio: inputPriceRatio = '', margin = 0.05, decimal = 0,
-} = {}, likecoin:any, arweavePrice:any ): any {
-  let priceRatio = inputPriceRatio;
-  if (!priceRatio) {
-    priceRatio = new BigNumber(arweavePrice.usd).dividedBy(likecoin.usd).toFixed();
-  }
-  // At least 1 LIKE for 1 AR
+async function getPriceRatioBigNumber () {
+  const { data } = await axios.get(COINGECKO_PRICE_API);
+  const { likecoin, arweave: arweavePrice } = data;
+  const priceRatio = new BigNumber(arweavePrice.usd).dividedBy(likecoin.usd).toFixed();
   const priceRatioBigNumber = BigNumber.max(priceRatio, 1);
+  return priceRatioBigNumber;
+}
+
+export function convertARPriceToLIKE(ar: ArweavePrice, {
+  priceRatioBigNumber = 1 as any, margin = 0.05, decimal = 0,
```

Comment:
`priceRatioBigNumber` should not have default value, calling this with value `1` would cause serious calculation mistake. It should call `getPriceRatioBigNumber` if one is not set